lib/sdr_client.rb

```diff
@@ -5,17 +5,14 @@
 require 'active_support'
 require 'active_support/core_ext'
 require 'cocina/models'
+require 'zeitwerk'
 
-require 'sdr_client/version'
-require 'sdr_client/unexpected_response'
-require 'sdr_client/deposit'
-require 'sdr_client/update'
-require 'sdr_client/credentials'
-require 'sdr_client/find'
-require 'sdr_client/login'
-require 'sdr_client/login_prompt'
-require 'sdr_client/connection'
-require 'sdr_client/background_job_results'
+loader = Zeitwerk::Loader.for_gem
+loader.inflector.inflect(
+  'md5' => 'MD5',
+  'sha1' => 'SHA1'
+)
+loader.setup
```

Comment:
Now we don't have to add manual require statements whenever we add a new class or rename a class to the client.